benchmarks/graphics_pipeline/main.cpp

```diff
@@ -116,13 +142,12 @@ class ProjApp
     grfx::DescriptorSetLayoutPtr                    mSetLayout;
     std::array<grfx::ShaderModulePtr, kAvailableVsShaders.size()> mVsShaders;
     std::array<grfx::ShaderModulePtr, kAvailablePsShaders.size()> mPsShaders;
+    std::array<Scene, kAvailableScenes.size()>                    mScenes;
     PerspCamera                                     mCamera;
     float3                                          mLightPosition = float3(10, 100, 10);
     BenchmarkSettings                               mBenchmarkSettings;
+    size_t                                                        mCurrentSceneIndex;
```

Comment:
Indentation looks off, I guess our formatting check didn't catch this

Reply:
Ah yes,thanks!

Reply:
Looks like the format was doing that on purpose. Maybe a mixed tab/space thing, changed the lines above so it looks consistent.